pkg/pluginmanager/manager.go

```diff
@@ -681,10 +691,19 @@ func DeletePlugin(options DeletePluginOptions) error {
 	return nil
 }
 
-// SyncPlugins automatically downloads all available plugins to users machine
+// SyncPlugins automatically installs all plugins of the current contexts
```

Comment:
nit: comment is only correct if the CR feature is enabled.

Reply:
What do you mean by this?  Do you mean, "if a CR exists for a plugin", or is there an actual feature-flag for this feature?  Maybe you are referring to the context-aware feature-flag?

Reply:
Sorry I should have been clearer. It looks like the comment only applies to the case where
`configlib.IsFeatureActivated(constants.FeatureCentralRepository)` is true. When false we install more than just plugins of the current contexts.

Reply:
Of course. Will update. Thanks!

Reply:
Oh, I understand why I was confused.  I interpreted "CR" to mean "custom resource" when you meant "Central Repository" 😄 .  We're going to have to be careful about that one 🤯